src/onefetch/cli.rs

```diff
@@ -228,6 +229,21 @@ impl Cli {
                 .takes_value(true)
                 .help("Ignore all files & directories matching EXCLUDE."),
             )
+        .arg(
+            Arg::with_name("format")
+                .short("f")
+                .long("format")
+                .help("Select a output format.")
+                .takes_value(true)
+                .default_value("human")
+                .validator(
+                    |t| {
+                        let opts = vec!["human", "json"];
+                        if opts.contains(&t.as_str()) {return Ok(()); }
+                        Err(String::from(format!("Output type must be one of the following: {:?}", opts)))
+                    }
+                )
```

Comment:
Instead of a validator, you might want to use `.possible_values`, which should accomplish roughly the same thing.